src/scripts.js

```diff
@@ -47,33 +84,27 @@ function viewFavorites() {
     user.favoriteRecipes.forEach(recipe => {
       cardArea.insertAdjacentHTML('afterbegin', `<div id='${recipe.id}'
       class='card'>
-      <header id='${recipe.id}' class='card-header'>
+      <header data-id='${recipe.id}' class='card-header'>
       <label for='add-button' class='hidden'>Click to add recipe</label>
-      <button id='${recipe.id}' aria-label='add-button' class='add-button card-button'>
-      <img id='${recipe.id}' class='add-image'
+      <button data-id='${recipe.id}' aria-label='add-button' class='recipe-to-cook add-button card-button'>
+      <img data-id='${recipe.id}' class='add-image recipe-to-cook'
```

Comment:
I'm so interested in the data-id attribute. I've never seen it before. It's cool to learn about it through your code!